scripts/toolchains/android.cmake

```
@@ -6,8 +6,13 @@ set(CMAKE_ANDROID_NDK_TOOLCHAIN_VERSION clang CACHE STRING "")
 
 if (VCPKG_TARGET_TRIPLET MATCHES "^arm64-android")
     set(ANDROID_ABI arm64-v8a CACHE STRING "")
+elseif(VCPKG_TARGET_TRIPLET MATCHES "^armv6-android")
+    set(ANDROID_ABI armeabi CACHE STRING "")
 elseif(VCPKG_TARGET_TRIPLET MATCHES "^arm-android")
     set(ANDROID_ABI armeabi-v7a CACHE STRING "")
+    if(DEFINED ENV{ANDROID_ARM_NEON})
+        set(ANDROID_ARM_NEON $ENV{ANDROID_ARM_NEON} CACHE STRING "")
+    endif()

```

```suggestion
elseif(VCPKG_TARGET_TRIPLET MATCHES "^arm-android")
    set(ANDROID_ABI armeabi-v7a CACHE STRING "")
elseif(VCPKG_TARGET_TRIPLET MATCHES "^armv7neon-android")
    set(ANDROID_ABI armeabi-v7a CACHE STRING "")
    set(ANDROID_ARM_NEON ON CACHE BOOL "")
```

https://developer.android.com/ndk/guides/cmake#android_abi

---

Then, also add an `armv7neon-android` community triplet.

---

can we do it better name, just arm-neon?

---

it can be neon with armv8 and etc.

---

I need to add arm-neon cmake file.

---

done